DRAFT_RELEASE_NOTES.md

```diff
@@ -12,8 +12,17 @@ _There are no breaking changes in the 1.10 release._
 _There are no known issues with the 1.10 release._
 
 # Known Vulnerabilities
-| Date<br/>identified | Vulnerability | Severity | Package | Affected <br/>versions | CVE | Fixed <br/>in |
-|---------------------|---------------|----------|---------|------------------------|-----|---------------|
+| Date<br/>identified | Vulnerability                                                             | Severity | Package                   | Affected <br/>versions | CVE                                                               | Fixed <br/>in |
+|---------------------|---------------------------------------------------------------------------|----------|---------------------------|------------------------|-------------------------------------------------------------------|---------------|
+| 09/25/2024          | Uncontrolled resource consumption in braces                               | High     | docs/antora-aissemble-ui/ | < 3.0.3                | [CVE-2024-4068](https://nvd.nist.gov/vuln/detail/CVE-2024-4068)   | 3.0.3         |
```

Comment:
S: This is just for our own docs and nothing that we package for re-use with other projects. I don't think it's appropriate to list the vulns here.

Reply:
Copy that, removed.